xlml/apis/task.py

```diff
@@ -418,6 +418,116 @@ def post_process(self) -> DAGNode:
       return group
 
 
+@dataclasses.dataclass
+class GpuXpkTask(BaseTask):
```

Comment:
Is there a functional difference between this and TpuXpkTask, or could we just make it a common Task between TPU and GPU?

Reply:
Good point! I think has some small params differences, i.e. in TPU we pass `tpu-type`, but for CPU and GPU, we pass `device-type`. Let's unify them as one `XpkTask`?

Reply:
Nice, `device-type` will work for TPUs as well.